Trappist/src/Promact.Trappist.Core/Controllers/QuestionController.cs

```diff
@@ -10,7 +13,7 @@ public class QuestionController : Controller
     {
         private readonly IQuestionRespository _questionsRepository;
 
-        public QuestionController(IQuestionRespository questionsRepository)
+        public QuestionController(IQuestionRespository questionsRepository, UserManager<ApplicationUser> userManager)
```

Comment:
I am using userManager to get ApplicationUser
Line: 60 
`question.ApplicationUser = await _userManager.FindByEmailAsync(userEmail);`